resources/hooks/local/pre-push

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+GIT_USER=$(git config user.name)
+GIT_EMAIL=$(git config user.email)
+COMMIT_MSG_FILE=$1
+
+# Fetch the GIT diff and format it as command input:
+DIFF=$(git diff -r -p -m -M --full-index --staged | cat)
```

Comment:
Ok I found the right one `GIT_CURRENT_BRANCH=$(git name-rev --name-only HEAD) && git diff --name-only origin/$GIT_CURRENT_BRANCH..HEAD --oneline` in that way get the actual branch